### ISSUE TITLE
Version conflict of tailwindcssui with webpack

### ISSUE THREAD

**Issue by snake65719** [tailwindlabs/tailwindui-issues#1570]
**What component (if applicable)**
- URL for category: https://tailwindui.com/components/application-ui/navigation/sidebar-navigation
- Component name: [With expandable sections](https://tailwindui.com/components/application-ui/navigation/sidebar-navigation#component-34180dd5b86e303286a8aa59b2ca9c34)

**Describe the bug**
A clear and concise description of what the bug is. TypeError: (0 , react__WEBPACK_IMPORTED_MODULE_0__.createContext) is not a function
    at eval (webpack-internal:///(rsc)/./node_modules/@headlessui/react/dist/internal/open-closed.js:9:75)
    at (rsc)/./node_modules/@headlessui/react/dist/internal/open-closed.js (D:\learn\testui\my-app\.next\server\vendor-chunks\@headlessui.js:110:1)     
    at __webpack_require__ (D:\learn\testui\my-app\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///(rsc)/./node_modules/@headlessui/react/dist/components/disclosure/disclosure.js:10:82)
    at (rsc)/./node_modules/@headlessui/react/dist/components/disclosure/disclosure.js (D:\learn\testui\my-app\.next\server\vendor-chunks\@headlessui.js:20:1)
    at __webpack_require__ (D:\learn\testui\my-app\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///(rsc)/./app/page.js:7:108)
    at (rsc)/./app/page.js (D:\learn\testui\my-app\.next\server\app\page.js:184:1)
    at Function.__webpack_require__ (D:\learn\testui\my-app\.next\server\webpack-runtime.js:33:42)
    at async e9 (D:\learn\testui\my-app\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:35:396554)
    at async rL (D:\learn\testui\my-app\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:7290)
    at async D:\learn\testui\my-app\node_modules\next\dist\compiled\next-server\app-page.runtime.dev.js:39:10360
    at async Promise.all (index 0)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Browser/Device (if applicable)**

**Additional context**

**Comment by snake65719** [comment]
with create-next-app

**Comment by RobinMalfait** [comment]
Hey!

This typically means that you have multiple versions of React installed. Can you create and share a minimal reproduction repo?

In the meantime, you can also remove the `.next` folder and `node_modules` and re-install dependencies using `npm install`. Once that is done, restart the dev server.

Additionally, you can use `npm ls react` which shows you a try where React is used (and which version).

**Comment by reinink** [comment]
Hey! Going to close this one since we haven't heard back, but feel free to reopen a new issue if you're able to provide a minimal reproduction of this 👍
